week3/2-website/main.css

```diff
@@ -0,0 +1,266 @@
+
```

Comment:
Try to keep spacing consistent through out the file. It makes it easier to read
Check for an extension on vscode that will format the file.